pages/mesosphere/dcos/services/beta-kafka/2.10.0-2.4.0-beta/quick-start/index.md

```diff
@@ -0,0 +1,131 @@
+---
```

Comment:
why are you creating a new folder `beta-kafka`? Shouldn't this be under the `/kafka/` folder? There is nothing in that folder currently unless we're creating a new beta-kafka framework?

Reply:
this is what beta-jupyter did currently, is this not the way to go?

Reply:
beta-jupyter was a brand new framework which later became DSE. Kafka is already well established as a DCOS framework and you are just releasing a version with `-beta` in the name, no?

Reply:
correct, we've already released on to Universe.

Reply:
if your aim is for the user to install `dcos install beta-kafka` then this makes sense but that is kind of strange... I assume you mean to do `dcos install kafka --version=xx-beta`

Reply:
I didn't really have a strong preference either way, have we discouraged use of `beta-kafka` in the past?

Reply:
oh you already released on universe, so the user _has_ to install via `dcos install beta-kafka`. Okay, nvm me. But should be noted the last time `beta-kafka` framework was worked on was 2 years ago... see [last revision](https://github.com/mesosphere/universe/tree/version-3.x/repo/packages/B/beta-kafka/8)

We discouraged it in the past simply from the standpoint of not confusing customers and because they will sometimes go into production with these FWs. And then we'd get back COPS tickets with beta- frameworks and 🤦 

question, why make it a beta release? Couldn't it simply have been version `2.10.0-2.4.0` and if you saw issues, do a patch release?

Reply:
this is a new feature release with Custom TLS certs, we wanted feedback from customers but this won't necessarily be the final product, need to expliclity flag it as `beta` so we're not backed into a corner supporting it.

Reply:
@samvantran  we are highlighting that this is a beta release and not suitable for production using 
`#include /mesosphere/dcos/services/include/beta-software-warning.tmpl`